.github/CODEOWNERS

```diff
@@ -72,6 +72,9 @@
 # PRLabel: %Cognitive - Form Recognizer
 /sdk/formrecognizer/                            @annelo-msft @kinelski @maririos
 
+# PRLabel: %Azure Monitor
```

Comment:
There is no "Azure Monitor" label in the repo

Reply:
There are a few others though
![image](https://user-images.githubusercontent.com/1697911/93530273-25e8a880-f8f2-11ea-972c-5cfd3501e3be.png)

Reply:
@rajkumar-rangaraj do we need to create a new label or any of the existing ones makes sense?

Reply:
@AlexGhiondea It's better if we could create a new label with name `Monitor - Exporter`

Reply:
Shall I create a new label and refer that in CODEOWNERS?